app/frontend/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -1,15 +1,22 @@
 import ErrorFallback from "components/ErrorFallback";
-import { ReactNode } from "react";
-import { ErrorBoundary as ReactErrorBoundary } from "react-error-boundary";
+import { PropsWithChildren } from "react";
+import {
+  ErrorBoundary as ReactErrorBoundary,
+  ErrorBoundaryProps,
+} from "react-error-boundary";
 import { useLocation } from "react-router";
 
-export default function ErrorBoundary({ children }: { children: ReactNode }) {
+export default function ErrorBoundary({
+  children,
+  ...otherProps
+}: PropsWithChildren<Partial<ErrorBoundaryProps>>) {
```

Comment:
Doesn't `ErrorBoundaryProps` already contain `children`? So maybe `PropsWithChildren` isn't needed.

Reply:
```javascript
declare type ErrorBoundaryProps = ErrorBoundaryPropsWithFallback | ErrorBoundaryPropsWithComponent | ErrorBoundaryPropsWithRender;
declare type ErrorBoundaryState = {
    error: Error | null;
};
declare class ErrorBoundary extends React.Component<React.PropsWithRef<React.PropsWithChildren<ErrorBoundaryProps>>, ErrorBoundaryState> {
```

Reply:
These props are also being wrapped by the ErrorBoundary component (the one from react-error-boundary) using `PropsWithChildren`. The type `ErrorBoundaryProps` itself doesn't have the children prop yet.